### ISSUE TITLE
After carefully reading the instructions, I found that it was noted that

### ISSUE THREAD

**Issue by MLTOL** [muschellij2/WhiteStripe#6]
After carefully reading the instructions, I found that it was noted that 

_This method performs white matter mean and standard deviation estimates on data that has been rigidly-registered to the MNI template and uses histogram-based methods._

I have not register to MNI template, I will try it.

_Originally posted by @chihyanghsu0805 in https://github.com/muschellij2/WhiteStripe/issues/5#issuecomment-293691312_

**Comment by MLTOL** [comment]
Hi,

I have run into the same error, although my images are registered to an MNI template.

I used the spm toolbox to register my images to an isotropic MNI template (MNI305) and develop a brain mask. This resulted in similar shaped histograms:

non-isotropic, non-registered:
![image](https://user-images.githubusercontent.com/30501164/53553060-11d92c80-3b3d-11e9-96a3-c1d2c2d98874.png)

dimension of image: 256/256/30

Isotropic, registered:
![image](https://user-images.githubusercontent.com/30501164/53553046-0980f180-3b3d-11e9-8d94-87a360708886.png)

dimension of image: 172/220/156

For the non-registered, non-isotropic images the Whitestripe normalisation works fine, for the registered isotropic image I get the error returned:

    **Error in gam(y ~ X - 1, paraPen = list(X = D), method = method, ...) : 
    Model has more coefficients than data**

My images need to be isotropic for further methods. 

Is there an explanation why I get the error returned?

**Comment by muschellij2** [comment]
Sorry to be unclear, but the MNI template is the MNI152 6th generation: http://nist.mni.mcgill.ca/?p=858.  This is also available in the MNITemplate package: https://github.com/Jfortin1/MNITemplate.  Can you register to that template and try again?

1) If you're registering to MNI152, you do not need to skull strip, but you can.
2) If you skull strip, please run `whitestripe(img, stripped = TRUE)`

**Comment by MLTOL** [comment]
Thank you for the fast reply. I used the new suggested MNI template, but unfortunately still the same error. Dimensions of the images now are 193/229/193, and the histogram:

![image](https://user-images.githubusercontent.com/30501164/53624611-2a127f80-3c01-11e9-80e3-838c0825a994.png)

My images are skull stripped. Background is set to 0.

**Comment by muschellij2** [comment]
Can you share your original image?  Also, I'm unsure as to why your MNI is 193/229/193 as the T1 should be 182x218x182
```r
x = MNITemplate::readMNI()
dim(x)
```
```
[1] 182 218 182
```

**Comment by MLTOL** [comment]
Thank you, I downloaded the new version of the MNI template from the given link: 

> http://nist.mni.mcgill.ca/?p=858

These templates have different dimensions than the templates on the gitHub library:

> https://github.com/Jfortin1/MNITemplate

I downloaded the template from the github library and the WhiteStripe library seems to work fine now.

Thank you for your help. I will let you know if I run into any other problems.
